third_party/closure-compiler/src_additional/chrome_extensions.js

```diff
@@ -126,3 +126,51 @@ chrome.certificateProvider.SignDigestRequestEvent.prototype.removeListener = fun
  * @type {!chrome.certificateProvider.SignDigestRequestEvent}
  */
 chrome.certificateProvider.onSignDigestRequested;
+
+/**
+ * @const
+ */
+chrome.loginState = {};
+
+/**
+ * @enum {string}
+ */
+chrome.loginState.ProfileType = {
+  SIGNIN_PROFILE: '',
```

Comment:
Thinking more on this, I think the string values should be non-empty, e.g.:

`  SIGNIN_PROFILE: 'SIGNIN_PROFILE',`

otherwise the `chrome.loginState.ProfileType.SIGNIN_PROFILE` expression would emit an empty string?